content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst

```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
-         in Odoo Accounting
+   :alt: Example of a reconciliation model that matches existing invoices and bills automatically
+         in Odoo Accounting.
 
 Configuration
 =============
 
-To manage or create new **Reconciliation Models**, go to :menuselection:`Accounting -->
-Reconciliation --> Reconciliation Models`. Alternatively, you can also open this menu from the
-Accounting Overview, by going to your Bank Journal card, clicking on the three little dots, and then
-on *Reconciliation Models*.
+To manage or create new **reconciliation models**, go to :menuselection:`Accounting dashboard -->
+Configuration --> Reconciliation Models`. Alternatively, you can also open this menu from
+:menuselection:`Accounting dashboard --> Bank --> Drop-down menu (⋮) --> Reconciliation Models`.
 
 .. image:: reconciliation_models/reconciliation_models_overview.png
    :align: center
-   :alt: Open the Reconciliation Model menu from the overview dashboard in Odoo Accounting
+   :alt: Open the reconciliation model menu from the overview dashboard in Odoo Accounting.
 
 .. important::
-   The first entry, named *Invoices Matching Rule*, is the one responsible for the current matching
-   of invoices and bills. Therefore, it is advised to leave it at the top of the list and not to
-   delete it.
+   The first entry, named :guilabel:`Invoices Matching Rule`, is the one responsible for the current
+   matching of invoices and bills. Therefore, it is advised to leave it at the top of the list and
+   not to delete it.
 
-Open the model you want to modify, or click on *Create* to create a new one, then fill out the form.
+Open the model you want to modify or click on :guilabel:`Create` to create a new one, and fill out
+the form.
 
 Type
 ----
 
 See :ref:`above <reconciliation_models_types>` for an explanation about the different types of
-Reconciliation Models.
+reconciliation models.
 
 .. note::
-   If the *Documents* application is installed on your database, an additional **Activity type**
-   field appears when *To check* is ticked. Selecting the value *Reconciliation request* implies
-   that, whenever you use this model, a *Request Document* window pops up to request a document from
-   a user.
+   If the *Documents* application is installed on your database, an additional
+   :guilabel:`Activity type` field appears when :guilabel:`To Check` is ticked. Selecting the value
+   `Reconciliation request` implies that whenever you use this model, a :guilabel:`Request Document`
+   window pops up to request a document from the user.
 
-Conditions on Bank Statement Line
+Conditions on bank statement line
 ---------------------------------
 
-Define here all the conditions that are required for a Reconciliation Model to be applied.
+Define here all the conditions you wish to apply for a reconciliation model to be applied. Depending
+on the :guilabel:`Type` of models you chose, different options are available.
```

Comment:
```suggestion
Define here all the conditions that should be fulfilled for the reconciliation model to be applied. Depending
on the :guilabel:`Type` of model you choose, different options are available.
```

> to apply for a reconciliation model to be applied

I would rephrase that. Feel free to use something better than my suggestion though ;)